tauri-app/src/routes/orders/add/+page.svelte

```diff
@@ -161,8 +212,46 @@
   </Helper>
 </div>
 
-<Button disabled={isCharting} on:click={chart}><span class="mr-2">Make charts</span>{#if isCharting}<Spinner size="5" />{/if}</Button>
-<Charts {chartData} />
+<Tabs
+  style="underline"
+  contentClass="mt-4"
+  defaultClass="flex flex-wrap space-x-2 rtl:space-x-reverse mt-4"
+>
+  <TabItem open title="Rainlang">
+    <div class="flex flex-col gap-y-2">
+      <Label>Select Scenario</Label>
+      {#if scenarios === undefined || Object.keys(scenarios).length === 0}
```

Comment:
i'm getting concerned about all the different ways we check for the lack of a value across this codebase, coupled with the lack of tests, and observance of real bugs in the wild due to this fact

can we defer to a lib like lodash to handle these subtle yet repetitive checks?

Reply:
Im not sure I understand exactly what you're asking, `scenarios`here  uses lodash `pickedBy()`which can result in undefined or empty object if there are no scenarios defined, and here then it is just checked if it's empty or undefined.